spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -86,6 +86,18 @@ def expect_retryable_error(error_class)
             end
           end
         end
+
+        it 'includes a proper classification code for EVSS submission' do
```

Comment:
Add a test for the negative case to make sure RRD is not setting the classification code for irrelevant claims.